lib/build/dispatcher.js

```diff
@@ -24,23 +24,43 @@ class Dispatcher {
     if (!repoConfig) {
       return
     }
+    if (payload.head_commit) {
```

Comment:
in changes below, we trigger a build job if the payload has a head_commit, else we trigger a cleanup job